src/app/api/user/login/route.js

```diff
@@ -11,7 +11,8 @@ export const POST = async request => {
 
     const user = await prisma.user.findUnique({ where: { username } })
     if (!user) return ERROR.INVALID_FIELDS()
-
+    if (!user.active) return ERROR.USER_INACTIVE()
```

Comment:
```suggestion
    if (!user.active) return ERROR.INVALID_FIELDS()
```
No es bueno Que seas tan explícito con los errores del login, porque es un endpoint público